### DISCUSSION TITLE
Sub-libraries / Auto add files to collections

### DISCUSSION THREAD

**Discussion by AnarchyCent** [gotson/komga#1475] [Ideas]
It would be great if libraries had sub-libraries. The files found in a subfolder would be displayed there. For example:

/data/books/Comics
├── Marvel
│   ├── **Spider-Man**
│   │   ├── Spider-Man_Adventures_001.cbz
│   │   ├── Spider-Man_Adventures_002.cbz
│   │   ├── Spider-Man_Adventures_003.cbz

Or if files were automatically added to collections, depending on the folder in which the files are located. So that you don't have to add them manually every time. However, it would be important to be able to change the automatic sorting by date.

**Comment by gotson** [comment]
What would sub-libraries achieve exactly ?

Series can already be automatically added to collections using embedded metadata.

**Reply by AnarchyCent** [reply]
Take a look at the ajslate/codex approach. If you have files with similar names but different numbering/date, they will be grouped automatically. So you don't have to set up folder logic and move the files manually.

Embedded metadata does not work with PDF files.

**Reply by gotson** [reply]
> Take a look at the ajslate/codex approach. If you have files with similar names but different numbering/date, they will be grouped automatically. So you don't have to set up folder logic and move the files manually.

still not clear to me.

> Embedded metadata does not work with PDF files.

Embedded metadata in PDF is known to be crap, that's why.

**Comment by gotson** [comment]
Use case not clear.
